app-multimedia/jack-capture/autobuild/build

```
@@ -0,0 +1,13 @@
+MAKE_DEF=(
+        PREFIX='/usr'
+)
+MAKE_AFTER=(
+        CFLAGS="$CFLAGS"
+        LDFLAGS="$LDFLAGS"
+)
+
+export LDFLAGS="${LDFLAGS} -latomic"
+make "${MAKE_DEF[@]}" "${MAKE_AFTER[@]}" -j$(nproc) SKIP_STRIPPING=true
```

```suggestion
abinfo "Building jack-capture ..."
make "${MAKE_DEF[@]}" "${MAKE_AFTER[@]}" -j$(nproc) SKIP_STRIPPING=true
```

Please add respective output here.